### PR TITLE
Docs wrapper

### PR DESCRIPTION
**Pull request by iAdramelk**
- DocsHead component.
- DocsNavigation component.
- Icons in config.
- Version switcher.

### CODE REVIEW

**Comment by iAdramelk** [comment]
@alex-kovoy sorry, created PR from master instead of previous branch.

File: components/DocNavigation/DocNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+const DocNavigationCategory = ({
+  id,
+  opened,
+  onToggleOpened,
+  icon,
+  title,
+  entries,
+}: DocNavigationCategoryProps) => {
+  const router = useRouter();
+  const toggleOpened = useCallback(() => onToggleOpened(opened ? null : id), [
```
Comment (alex-kovoy) [inline components/DocNavigation/DocNavigation.tsx:36]: this optimization `useCallback` is unnecessary. Also consider moving the Category to a separate file (

```
/DocNavigation/
   DocNavigation.tsx
   DocNavigationCategory.tsx
   index.ts
```

File: components/Icon/icons.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export { default as bolt } from "./svg/bolt.svg";
```
Comment (alex-kovoy) [inline components/Icon/icons.ts:1]: lets move it under `svg` folder and rename it to `index.ts`

`index.ts` is a way to access package elements from javascript. The fact that these elements can represent an `svg` file is an implementation detail handled by webpack. So it's ok to use the same index-accessing in this case as well.

```ts
import { book, code, flag } from './svg';
```

**Review by alex-kovoy — APPROVED** [review]
with some minor comments

**Pull request merged**